core/value.ml

```diff
@@ -6,6 +6,9 @@ open Var
 
 let _ = ParseSettings.config_file
 
+let internal_error message =
+  raise (Errors.internal_error ~filename:"instantiate.ml" ~message)
```

Comment:
I think this might be a C&P error (`instantiate` --> `value`)

Reply:
Yup, thanks for catching.